include/faabric/scheduler/MpiWorld.h

```diff
@@ -221,6 +214,24 @@ class MpiWorld
                                                               int recvRank);
     void closeMpiMessageEndpoints();
 
+    // Support for asyncrhonous communications
+    std::shared_ptr<faabric::scheduler::MpiMessageBuffer>
```

Comment:
Could perhaps save a few characters by removing the `faabric::scheduler::` prefix when in the `faabric::scheduler` namespace already.